manager/controllers/app/modules/moduleinterface.go

```diff
@@ -17,22 +17,30 @@ type Transformations struct {
 	Allowed            bool
 	EnforcementActions []pb.EnforcementAction
 	Message            string
+	// geography relevant for the read/write operation
+	// indicates where the workload runs (for read access) or where to write the data to (for copying data to another location)
+	Geo string
 }
 
 // DataInfo defines all the information about the given data set
+// TODO: Add support for recurrence in modules
 type DataInfo struct {
 	// Data asset unique identifier, not necessarily the same string appearing in the resource definition
 	AssetID string
-	// Processing geography
-	Geo string
 	// Application interface
```

Comment:
If there are no transformations, then we don't know the processing Geo?

Reply:
Workload geography is stored in the geo field of the READ action structure. It's where we want to read the data.